### PR TITLE
Cleaning app.py

### PR DESCRIPTION
**Pull request by TreyHeaney**
Final before deployment

### CODE REVIEW

File: spotify/app.py [hunk 1]
```diff
@@ -10,215 +10,117 @@
...
-                                                     time_range="short_term"
-                                                    )["items"]
-        # Raise error if no recent top tracks.
-        if not user_top_tracks:
-            return render_template("error.html", type='Empty Spotify')
+# Redirects users to login with spotify.
+@app.route("/spotify_login")
+def authorization():
+    sp = create_spotify_oauth()
+    auth_url = sp.get_authorize_url()
```
Comment (afroroboticist) [inline spotify/app.py:41]: What happens when user doesn't have an acount?

File: spotify/__init__.py [hunk 2]
```diff
@@ -1,3 +1 @@
-from spotify.app import create_app
-
-APP = create_app()
+from spotify.app import app
```
Comment (afroroboticist) [inline spotify/__init__.py:1]: Why does init.py only have the one import statement?
Just feels a bit under utilized to me.

**Review by afroroboticist — COMMENTED** [review]
Left you a few comments.

File: spotify/app.py (hunk 1 above)
Comment (TreyHeaney) [reply spotify/app.py:41]: Users are redirected to verify with spotify and if they don't have a spotify account they have to make one.

File: spotify/__init__.py (hunk 2 above)
Comment (TreyHeaney) [reply spotify/__init__.py:1]: __init__.py  Is actually technically worthless in our case. We wrapped the project in a module so the additional py files we import from are easier to manage and we don't get circular imports (ie an import importing something which imports that which imports it) and __init__.py  is a formality with this package structure. It's standard.

**Pull request merged**